Codespace Error - Error whith PNPM version while building codespace environment

### Which @ngrx/* package(s) are the source of the bug?

signals

### Minimal reproduction of the bug/regression with instructions

Hi, I'm trying to create the required codespace to create a feature for the NGRX Signal store and create a pull request but when building the codespace i get an error, I saw the error began at line: 693 with PNPM install.

[creation.log](https://github.com/user-attachments/files/18767798/creation.log)

Here is an extract of the log:


```
2025-02-12 12:19:51.924Z: pnpm install
2025-02-12 12:19:52.576Z:  ERR_PNPM_BAD_PM_VERSION  This project is configured to use v9.10.0 of pnpm. Your current pnpm is v9.1.0
 
If you want to bypass this version check, you can set the "package-manager-strict" configuration to "false" or set the "COREPACK_ENABLE_STRICT" environment variable to "0"
2025-02-12 12:19:52.591Z: postCreateCommand failed with exit code 1. Skipping any further user-provided commands.
 
2025-02-12 12:19:52.594Z: Error: Command failed: /bin/sh -c pnpm install
2025-02-12 12:19:52.595Z:     at b7 (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:235:130)
2025-02-12 12:19:52.595Z: {"outcome":"error","message":"Command failed: /bin/sh -c pnpm install","description":"The postCreateCommand in the devcontainer.json failed.","containerId":"aa415374e29d0a9de2464c4aaabb710d85557304fb59276edc70c2c0278aef75"}
2025-02-12 12:19:52.595Z:     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-02-12 12:19:52.595Z:     at async Am (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:4483)
2025-02-12 12:19:52.596Z:     at async Xw (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:3828)
2025-02-12 12:19:52.597Z:     at async $w (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:3032)
2025-02-12 12:19:52.597Z:     at async fa (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:227:2438)
2025-02-12 12:19:52.598Z:     at async mtA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:1534)
2025-02-12 12:19:52.599Z:     at async NH (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:465:964)
2025-02-12 12:19:52.600Z:     at async YtA (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:482:3771)
2025-02-12 12:19:52.601Z:     at async eB (/.codespaces/agent/bin/node_modules/@devcontainers/cli/dist/spec-node/devContainersSpecCLI.js:482:4886)
2025-02-12 12:19:52.602Z: devcontainer process exited with exit code 1
 
====================================== ERROR ====================================
2025-02-12 12:19:52.607Z: Failed to create container.
=================================================================================
2025-02-12 12:19:52.607Z: Error: Command failed: /bin/sh -c pnpm install
2025-02-12 12:19:52.610Z: Error code: 1302 (UnifiedContainersErrorFatalCreatingContainer)
 
====================================== ERROR ====================================
2025-02-12 12:19:52.619Z: Container creation failed.
=================================================================================
```

### Expected behavior

I expect the building phase pf the Codespace not to throw any error and PNPM install to work.

### Versions of NgRx, Angular, Node, affected browser(s) and operating system(s)

None

### Other information

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [x] No

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -13,7 +13,7 @@
   // "forwardPorts": [],
 
   // Use 'postCreateCommand' to run commands after the container is created.
-  "postCreateCommand": "pnpm install",
+  "postCreateCommand": "npm i pnpm -g && pnpm install",
   "waitFor": "postCreateCommand",
   "onCreateCommand": "sudo cp .devcontainer/welcome-message.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt",
 

